packages/amplication-data-service-generator/src/server/resource/controller/to-many.template.ts

```diff
@@ -7,16 +7,21 @@ import * as defaultAuthGuard from "../auth/defaultAuth.guard";
 // @ts-ignore
 import * as abacUtil from "../auth/abac.util";
 import { Request } from "express";
+// @ts-ignore
+import {ApiNestedQuery} from "../../decorators/api-nested-query.decorator";
+import {plainToClass} from "class-transformer";
 
 declare interface WHERE_UNIQUE_INPUT {
   id: string;
 }
 declare interface RELATED_ENTITY_WHERE_UNIQUE_INPUT {}
-declare interface RELATED_ENTITY_WHERE_INPUT {}
+declare class RELATED_ENTITY_WHERE_INPUT {}
```

Comment:
This class is not being used and can be removed

Reply:
This is weird. It's class on my file
![image](https://user-images.githubusercontent.com/39680385/152502350-0a090c5d-290a-4307-97e2-410e791a8d34.png)

@yuval-hazaz

Reply:
What should I do with line 28?
![image](https://user-images.githubusercontent.com/39680385/152503565-bd862e04-9176-4b2e-8da8-2e6c20c185ff.png)

Reply:
you are correct, my mistake 👍